packages/@react-spectrum/actiongroup/src/ActionGroup.tsx

```diff
@@ -462,7 +462,7 @@ function ActionGroupMenu<T>({state, isDisabled, isEmphasized, staticColor, items
             }
             isDisabled={isDisabled}
             staticColor={staticColor}>
-            {summaryIcon ? <More /> : null}
+            {summaryIcon || <More />}
```

Comment:
I know this reversion works. Did you confirm that the typescript strict mode fixes still pass with this change?

Reply:
Yes, for `yarn tsc-strict` got:

``` cmd
🎯 Found 189 strict files
🎉 All files passed
```